src/client/App/actions.js

```diff
@@ -0,0 +1,18 @@
+import { ENDPOINT_URL } from '../common/variables';
+
+// Action types
+export const FETCH_FAILED = 'FETCH_FAILED';
+export const FETCH_ITEMS = 'FETCH_ITEMS';
+
+// Action creators
+export const fetchItemsError = err => ({ type: FETCH_FAILED, err });
+export const recieveItems = json => ({ type: FETCH_ITEMS, items: json });
+
+// Dispatchers
+export const fetchItems = () => dispatch =>
+  fetch(`${ENDPOINT_URL}/items`, { credentials: 'same-origin' })
+    .then(resp => resp.json())
+    .then(json => {
+      setTimeout(() => dispatch(recieveItems(json)), 1500);
```

Comment:
This is only to simulate fetching/downloading experience for the user. 
Otherwise fetching is executed so fast that there is no preloader effect visible for the user.

Reply:
user don't need to see this if he/she got fast network :) for testing such stuff please use dev tools/network in chrome